config/default.json

```diff
@@ -14,6 +14,10 @@
   "readyRetryIntervalMS": 300000,
   "requireAuth": false,
   "schedulerIntervalMS": 300000,
+  "carStorage": {
+    "mode": "inmemory",
+    "s3BucketName": "myS3Bucket"
```

Comment:
Can also use the default name suggested below.

Is it possible to parameterize this and use something like `ceramic-${env}-cas` here?